modules/service/deployment/0.1/facets.yaml

```
@@ -401,11 +398,8 @@ spec:
             liveness_exec_command:
               type: array
               title: Liveness Exec Command
-              pattern: '^([^,]+)(,\s*[^,]+)*$'
-              x-ui-placeholder: "Enter readiness exec for the Pod, commands are separated
-                by commas"
-              x-ui-error-message: "Value doesn't match pattern, commands needs to
-                be comma separated"
+              x-ui-placeholder: "Add each command argument as a new entry" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)